allenact/algorithms/onpolicy_sync/losses/a2cacktr.py

```diff
@@ -53,6 +53,11 @@ def loss_per_step(  # type: ignore
         )
 
         dist_entropy: torch.FloatTensor = actor_critic_output.distributions.entropy()
+        dist_entropy = dist_entropy.view(
```

Comment:
I'm not sure I understand the need to match the dimensions of entropy and log_probs (as far as I understand, both are reduced to a scalr before being added).

Reply:
This is mostly unnecessary, it's just that this `def loss_per_step(...)` function returns everything before doing reduction and if other losses want to use these pre-reduction outputs (e.g. ADVISOR-type losses), it's a bit easier to do so if the dimensions match.

Reply:
I agree.